result/neighborsv4.yaml

```diff
@@ -0,0 +1,43 @@
+openapi: 3.1.0
+
+info:
+  title: IPv4 Neighbors models
+  version: ^0.0.0
+
+components:
+  schemas:
+    Neighborsv4.States.Request:
+      description: >-
+        The request to retrieve IPv4 Neighbor state (ARP cache entries) for device network interface(s).
+      type: object
+      properties:
+        ethernet_names:
+          description: >-
+            The names of device Ethernet interfaces for which Neighbors will be retrieved.
+            If no names are specified then the results will contain Neighbors for all Ethernet interfaces.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.Ethernet/properties/name"
+
+    Neighborsv4.State:
+      description: >-
+        IPv4 Neighbor state (ARP cache entry).
+      type: object
+      required: [ethernet_name, ipv4_address]
+      properties:
+        ethernet_name:
+          description: >-
+            The name of the device Ethernet interface associated with the neighbor.
```

Comment:
The name of the Ethernet interface associated with the Neighbor state (ARP cache entries)